var/spack/repos/builtin/packages/veloc/package.py

```diff
@@ -17,7 +17,8 @@ class Veloc(CMakePackage):
     tags = ['ecp']
 
     version('master', branch='master')
-    version('1.1',    sha256='2bbdacf3e0ce4e7c9e360874d8d85b405525bdc7bd992bdb1f1ba49218072160', preferred=True)
+    version('1.4', tag='veloc-1.4', preferred=True)
```

Comment:
Is there a reason to use a tag (which is a moving target) instead of a release zip file like the other versions? Also, `preferred=True` isn't needed here, Spack already defaults to the latest stable release.

Reply:
I'll look for a release zip and swap that for this.